### PR TITLE
Update pallet_xcm benchmark

### PR DESCRIPTION
**Pull request by zjb0807**
Closes: #2744

### CODE REVIEW

File: runtime/acala/src/lib.rs [hunk 1]
```diff
@@ -2031,6 +2031,10 @@ mod benches {
 		[module_liquid_crowdloan, benchmarking::liquid_crowdloan]
 		[module_nominees_election, benchmarking::nominees_election]
 	);
+	// frame_benchmarking::define_benchmarks!(
+	// 	// XCM
+	// 	[pallet_xcm, PalletXcmExtrinsicsBenchmark::<Runtime>]
+	// );
```
Comment (zjb0807) [inline runtime/acala/src/lib.rs:2037]: enable this after all of the benchmarks migrated to `frame-benchmarking`

**Review by xlc — APPROVED** [review]

**Pull request merged**